com.kms.cura.dal/src/com/kms/cura/dal/database/DatabaseHelper.java

```diff
@@ -140,4 +140,26 @@ public void insertReferenceRowsToReferenceTable(String database, List<ReferenceT
 			}
 		}
 	}
+	
+	public Entity queryEmailPasswordJoin(String tableName1, String tableName2, String email, String password, String onTable1, String onTable2) throws SQLException, ClassNotFoundException{
```

Comment:
I don't think it should be in UserDatabaseHelper because I do not search an user, I search for a patient or doctor so I implement it here and I use it in DoctorDatabaseHelper and PatientDatabaseHelper